app.rb

```diff
@@ -28,6 +28,7 @@
 class TreeStats < Sinatra::Base
   configure :production do
     use Sentry::Rack::CaptureExceptions
+    use RateLimiter, limit: 100, seconds: 60
```

Comment:
This means 100 requests are allowed every 60 seconds per IP address.
These numbers aren't based on any real data.

We could configure by environment variable.